helpers.php

```diff
@@ -293,6 +384,13 @@ function getAltapayOrderDetails($orderID)
     return Db::getInstance()->executeS($sql);
 }
 
+function getAltapayChildOrderDetails($orderID)
+{
+    $sql = 'SELECT * FROM `' . _DB_PREFIX_ . 'altapay_child_order` WHERE id_order =' . (int) $orderID;
```

Comment:
Same comments as above

Reply:
@emicha , I'm not sure why the code is not getting outdated but this has been handled

[https://github.com/AltaPay/plugin-prestashop/blob/update-wiki-paybylink/helpers.php#L388](https://github.com/AltaPay/plugin-prestashop/blob/update-wiki-paybylink/helpers.php#L388)

Reply:
We have added a validation for non-zero positive integer ✅

Reply:
The code is still showing a casting

![image](https://github.com/AltaPay/plugin-prestashop/assets/1629945/2ea91a75-2a82-45fb-8611-5d674aa27c45)

Reply:
Done